atm.js

```diff
@@ -1,21 +1,96 @@
+// * [Adding or removing a CSS class using JS](https://developer.mozilla.org/en-US/docs/Web/API/Element.classList)
+// * [Changing HTML with JavaScript](https://developer.mozilla.org/en-US/docs/Web/API/Element.innerHTML)
 
-var checkingDeposit = function(amount){
-  checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+var balance = 0;
+var balanceSaving = 0;
 
-}
+document.getElementsByClassName("balance")[0].className += " zero";
+document.getElementsByClassName("balance")[1].className += " zero";
 
-function checkingWithdrawal(amount){
+var checkingBalance = document.getElementsByClassName("balance")[0];
+var savingBalance =  document.getElementsByClassName("balance")[1];
 
+//
+var checkingDeposit = function(amount){
+  balance = balance + parseInt(amount);
+  checkingBalance.innerHTML = ("$" + balance);
+}
+//
+var checkingWithdrawal = function(amount){
+  if (balance >= parseInt(amount)){
+    balance = balance - parseInt(amount);
+    checkingBalance.innerHTML = ("$" + balance);
+  } else if ((balance + balanceSaving) > parseInt(amount)){
+    checkingBalance.innerHTML = ("$" + 0);
+    balanceSaving = ((balanceSaving + balance) - parseInt(amount));
+    savingBalance.innerHTML = ("$" + balanceSaving);
+    balance = 0;
+  } else {
+    checkingBalance.innerHTML = ("$" + balance);
+  }
+}
+//
+var savingsDeposit = function(amount){
+  balanceSaving = balanceSaving + parseInt(amount);
+  savingBalance.innerHTML = ("$" + balanceSaving);
 }
 
-function savingsDeposit(amount){
+var savingsWithdrawal = function(amount){
+  while ((balanceSaving - parseInt(amount)) >= 0){
+  balanceSaving = balanceSaving - parseInt(amount);
+  savingBalance.innerHTML = ("$" + balanceSaving);
+  }
+  }
 
-}
+//Event Handlers for Input Boxes
 
-function savingsWithdrawal(amount){
+checkingDepositBox = document.getElementsByTagName("input")[1].addEventListener("click", function(){
+  checkingDepositValue = document.getElementsByTagName('input')[0].value;
+  checkingDeposit(checkingDepositValue);
+  console.log("I'm listening");
+})
 
-}
+checkingWithdrawalBox = document.getElementsByTagName("input")[2].addEventListener("click", function(){
+  checkingWithdrawalValue = document.getElementsByTagName('input')[0].value;
+  checkingWithdrawal(checkingWithdrawalValue);
+  console.log("I'm listening");
+})
+
+checkingDepositBox = document.getElementsByTagName("input")[4].addEventListener("click", function(){
+  savingDepositValue = document.getElementsByTagName('input')[3].value;
+  savingsDeposit(savingDepositValue);
+  console.log("I'm listening");
+})
 
-// an eventListerner for each button, each one a "click"
\ No newline at end of file
+checkingWithdrawalBox = document.getElementsByTagName("input")[5].addEventListener("click", function(){
+  savingWithdrawalValue = document.getElementsByTagName('input')[3].value;
+  savingsWithdrawal(savingWithdrawalValue);
+  console.log("I'm listening");
+})
```

Comment:
I like that you're using console.logs to debug and figure out what things are happening in your code execution. However, I would use more semantic messages. Have the console.log say exactly whats happening when the console.log is happening